src/detect_entities.py

```diff
@@ -1,33 +1,41 @@
-from typing import List, TypedDict
+from typing import List
 import spacy
+from spacy.tokens import Span
 from src.entity_utils import MarkedEntity, is_entity_contained
 
-
 nlp = spacy.load("en_core_web_lg")
 
 
+def split_person_entity(entity: Span, source: str) -> List[MarkedEntity]:
+    split_entity: List[MarkedEntity] = []
+    ent_parts = entity.text.split(" ")
```

Comment:
minor:

```suggestion
    ent_parts: List[str] = entity.text.split(" ")
```

just think it's makes the rest of the function easier to scan.